sdk/storage/azure-storage-internal-avro/src/main/java/com/azure/storage/internal/avro/implementation/util/AvroUtils.java

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.internal.avro.implementation.util;
+
+import java.nio.ByteBuffer;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicInteger;
+
+/**
+ * Utility methods for Avro.
+ */
+public class AvroUtils {
```

Comment:
I'd rather break it down into more concrete bags of functionality. Otherwise it just becomes a bag do dump totally unrelated code and becomes a mess over time.

Reference: http://ralin.io/blog/oop-anti-patterns-utility-or-helper-classes.html

Reply:
https://github.com/Azure/azure-sdk-for-java/issues/10463